### DISCUSSION TITLE
Support pomsky in addition to regex

### DISCUSSION THREAD

**Discussion by yerlaser** [nushell/nushell#10501] [Ideas]
https://pomsky-lang.org/ is a thin layer over regex which makes the expressions much more readable, maintainable and easier to write.
Would be good if commands that accept regex (e.g. str and glob) also accepted this dialect via a switch or a global config.

**Comment by fdncred** [comment]
I'd be interested in seeing how this would work. We could experiment with the `parse` command or with an nushell plugin.

**Reply by yerlaser** [reply]
I currently use `pomsky` CLI within parenthesis and wrap the expression in back ticks like so:
```
find -r (pomsky -n `'a'+ 'b'`)
```
Works quite well. Might create an alias to reduce typing.
So, it appears that no special integration is necessary after all.
